as/src/fabric/clustering.c

```diff
@@ -1336,21 +1336,6 @@ if (((size) > STACK_ALLOC_LIMIT()) && buffer) {cf_free(buffer);}
 #define DEBUG(format, ...) cf_debug(AS_CLUSTERING, format, ##__VA_ARGS__)
 #define DETAIL(format, ...) cf_detail(AS_CLUSTERING, format, ##__VA_ARGS__)
 
-#ifdef TRACE_ENABLED
```

Comment:
bad commit message. unclear why you've removed this. Couldn't you just build without `TRACE_ENABLED` being defined.